### DISCUSSION TITLE
Atom runs scripts in /etc/pfofile.d

### DISCUSSION THREAD

**Discussion by cyclopticnerve** [atom/atom#22428] [General]
Can someone please confirm this for me? I am developing a project that puts a script into /etc/profile.d (scripts that run at login).
The script runs fine when I log in. But the script runs again when I open Atom. I am on elemetaryOS 5.1 Hera. I don't have any terminal plugins installed. Atom --version shows :
Atom    : 1.55.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
(Note that the About menu item does not work either...)

**Comment by icecream17** [comment]
You might want to update atom.
But before that, open atom in dev and safe mode (`atom --dev --safe`), then open the console (`ctrl+shift+i`) and try to reproduce your problem. If any errors pop up, a screenshot might be helpful to fix it.
If the problem still happens after updating you can still do the same thing.

**Comment by cyclopticnerve** [comment]
Can confirm that (atom --dev --safe) does not run the script. I'm 
guessing that the console i had previously installed is not completely 
uninstalled and is still trying to start a login console. Web dev 
console (ctrl+shift+i) is useless as it only happens at startup. Will 
uninstall/reinstall to see if that fixes it.
